Apps/frontend/src/const.js

```diff
@@ -41,7 +41,7 @@ export const MIN_AMPLITUDE = 0.0;
 export const MAX_AMPLITUDE = NUM_INTERVALS_HORIZONTAL / 2;
 export const WAVE_CURSOR_SIZE = 50;
 
-export const MIN_CONTROL_PANEL_BOTTOM_HEIGHT = 280; //px
+export const MIN_CONTROL_PANEL_BOTTOM_HEIGHT = 150; //px
```

Comment:
At least for me the accordion button does no longer appear at the correct moment when zooming is changed due to this change.

Reply:
Reversed